cmd/configure.go

```diff
@@ -73,14 +75,19 @@ var openVPNConfigCmd = &cobra.Command{
 	},
 }
 
-// openVPNConfigureCmd represents the `furyagent configure openvpn` command
+// openVPNClientConfigureCmd represents the `furyagent configure openvpn-client` command
 var openVPNClientConfigCmd = &cobra.Command{
 	Use:   "openvpn-client",
 	Short: "Get OpenVPN users client from s3",
```

Comment:
I think we should update the description because now you can also revoke clients inside this command

Reply:
Updated description.